index.coffee

```diff
@@ -106,14 +113,22 @@ class ProgressTracker
 class ProgressReporter
 	constructor: (@renderProgress) -> #
 
-	# Create a stream that transforms docker daemon's onProgress
-	# events to include total progress metrics.
+	checkProgressError: (error, extraInfo) ->
+		prefix = "Progress error: [#{extraInfo}]"
+		console.warn(prefix, error.message ? error)
+		if error.message?.indexOf('unknown blob') >= 0
+			error.message = "#{prefix} #{error.message}"
+			throw error
```

Comment:
This is relatively conservative change that only throws 'unknown blob' errors, not any other errors. I considered throwing every error, but I don't really have data about the possible range of errors and if they all should abort the creation of the release. Previously, no errors were thrown at all.